test/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "minimal-jwt-test-project",
```

Comment:
Do we need this separate package file? Seems odd. I wonder if this is a mistake in minimal-cipher as well ... and if not, what we need it for.

Reply:
Fixed.